libdnf.spec

```diff
@@ -13,7 +13,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.0
+Version:        0.7.1
```

Comment:
I would like to keep the change, because it is there for good reason. I introduce the new feature that will be required by dnf, therefore I have to increase the version to allow specific require in DNF. The change should prevent possibility of incompatible combination released DNF and libdnf. You can argue that this can be solved during release, but according to my experience the requirement of increase in version in required component can be easily overlooked during release process, but if it is covered already in upstream, the chance of producing incompatibility is getting lower. Also I have to point out that the person who do the release is often not the same that introduce or review the incompatible change. I also would like to introduce increasing the version every time when the new feature will be required by other package, especially if it can cause problems, as a common practice for libdnf stack. The down site of it is that minor version will be increased more often and not all version will be released. I will be very grateful if all admins of libdnf can place their opinion on version change.